$NetBSD: patch-src_wswan_dis_opcodes_inc,v 1.1 2012/05/21 19:37:26 dholland Exp $

Comment out unused static data that refers to nonexistent functions.
Sometimes these elements are dropped by the compiler and sometimes
they fail the build. It is not at all clear why the behavior varies.

--- src/wswan/dis/opcodes.inc~	2009-05-15 01:30:50.000000000 +0000
+++ src/wswan/dis/opcodes.inc
@@ -37,7 +37,7 @@ Ia_andw_Gw_Ew = { "and", "andw", Gw, Ew,
 
 Ia_boundw_Gw_Ma = { "bound", "boundw", Gw, Ma, XX },
 
-Ia_call_Ed = { "call", "call", Ed, XX, XX },
+//Ia_call_Ed = { "call", "call", Ed, XX, XX },
 Ia_call_Ew = { "call", "call", Ew, XX, XX },
 Ia_call_Jd = { "call", "call", Jd, XX, XX },
 Ia_call_Jw = { "call", "call", Jw, XX, XX },
@@ -111,8 +111,8 @@ Ia_jl_Jw = { "jl", "jl", Jw, XX, XX },
 Ia_jle_Jb = { "jle", "jle", Jb, XX, XX },
 Ia_jle_Jd = { "jle", "jle", Jd, XX, XX },
 Ia_jle_Jw = { "jle", "jle", Jw, XX, XX },
-Ia_jmp_Ed = { "jmp", "jmp", Ed, XX, XX },
-Ia_jmp_Eq = { "jmp", "jmp", Eq, XX, XX },
+//Ia_jmp_Ed = { "jmp", "jmp", Ed, XX, XX },
+//Ia_jmp_Eq = { "jmp", "jmp", Eq, XX, XX },
 Ia_jmp_Ew = { "jmp", "jmp", Ew, XX, XX },
 Ia_jmp_Jb = { "jmp", "jmp", Jb, XX, XX },
 Ia_jmp_Jd = { "jmp", "jmp", Jd, XX, XX },
@@ -159,11 +159,11 @@ Ia_larw_Gw_Ew = { "lar", "larw", Gw, Ew,
 Ia_lcall_Apd = { "call far", "lcall", Apd, XX, XX },
 Ia_lcall_Apw = { "call far", "lcall", Apw, XX, XX },
 Ia_lcall_Mp = { "call far", "lcall", Mp, XX, XX },
-Ia_ldsl_Gd_Mp = { "lds", "ldsl", Gd, Mp, XX },
+//Ia_ldsl_Gd_Mp = { "lds", "ldsl", Gd, Mp, XX },
 Ia_ldsw_Gw_Mp = { "lds", "ldsw", Gw, Mp, XX },
 Ia_leave = { "leave", "leave", XX, XX, XX },
 Ia_leaw_Gw_Mw = { "lea", "leaw", Gw, Mw, XX },
-Ia_lesl_Gd_Mp = { "les", "lesl", Gd, Mp, XX },
+//Ia_lesl_Gd_Mp = { "les", "lesl", Gd, Mp, XX },
 Ia_lesw_Gw_Mp = { "les", "lesw", Gw, Mp, XX },
 Ia_lfence = { "lfence", "lfence", XX, XX, XX },
 Ia_lfsw_Gw_Mp = { "lfs", "lfsw", Gw, Mp, XX },
@@ -194,15 +194,15 @@ Ia_movb_Ob_AL = { "mov", "movb", Ob, AL,
 Ia_movb_R8_Ib = { "mov", "movb", R8, Ib, XX },
 
 Ia_movsb_Yb_Xb = { "movsb", "movsb", Yb, Xb, XX },
-Ia_movsbl_Gd_Eb = { "movsx", "movsbl", Gd, Eb, XX },
-Ia_movsbq_Gq_Eb = { "movsx", "movsbq", Gq, Eb, XX },
+//Ia_movsbl_Gd_Eb = { "movsx", "movsbl", Gd, Eb, XX },
+//Ia_movsbq_Gq_Eb = { "movsx", "movsbq", Gq, Eb, XX },
 Ia_movsbw_Gw_Eb = { "movsx", "movsbw", Gw, Eb, XX },
 Ia_movsl_Yd_Xd = { "movsd", "movsl", Yd, Xd, XX },
-Ia_movslq_Gq_Ed = { "movsxd", "movslq", Gq, Ed, XX },
+//Ia_movslq_Gq_Ed = { "movsxd", "movslq", Gq, Ed, XX },
 Ia_movsq_Yq_Xq = { "movsq", "movsq", Yq, Xq, XX },
 Ia_movsw_Yw_Xw = { "movsw", "movsw", Yw, Xw, XX },
-Ia_movswl_Gd_Ew = { "movsx", "movswl", Gd, Ew, XX },
-Ia_movswq_Gq_Ew = { "movsx", "movswq", Gq, Ew, XX },
+//Ia_movswl_Gd_Ew = { "movsx", "movswl", Gd, Ew, XX },
+//Ia_movswq_Gq_Ew = { "movsx", "movswq", Gq, Ew, XX },
 Ia_movw_AX_Ow = { "mov", "movw", AX, Ow, XX },
 Ia_movw_Ew_Gw = { "mov", "movw", Ew, Gw, XX },
 Ia_movw_Ew_Iw = { "mov", "movw", Ew, Iw, XX },
@@ -211,23 +211,23 @@ Ia_movw_Gw_Ew = { "mov", "movw", Gw, Ew,
 Ia_movw_Ow_AX = { "mov", "movw", Ow, AX, XX },
 Ia_movw_RX_Iw = { "mov", "movw", RX, Iw, XX },
 Ia_movw_Sw_Ew = { "mov", "movw", Sw, Ew, XX },
-Ia_movzbl_Gd_Eb = { "movzx", "movzbl", Gd, Eb, XX },
-Ia_movzbq_Gq_Eb = { "movzx", "movzbq", Gq, Eb, XX },
+//Ia_movzbl_Gd_Eb = { "movzx", "movzbl", Gd, Eb, XX },
+//Ia_movzbq_Gq_Eb = { "movzx", "movzbq", Gq, Eb, XX },
 Ia_movzbw_Gw_Eb = { "movzx", "movzbw", Gw, Eb, XX },
-Ia_movzwl_Gd_Ew = { "movzx", "movzwl", Gd, Ew, XX },
-Ia_movzwq_Gq_Ew = { "movzx", "movzwq", Gq, Ew, XX },
+//Ia_movzwl_Gd_Ew = { "movzx", "movzwl", Gd, Ew, XX },
+//Ia_movzwq_Gq_Ew = { "movzx", "movzwq", Gq, Ew, XX },
 Ia_mulb_AL_Eb = { "mul", "mulb", AL, Eb, XX },
 Ia_multibyte_nop = { "multibyte nop", "multibyte nop", XX, XX, XX },
 Ia_mulw_AX_Ew = { "mul", "mulw", AX, Ew, XX },
 Ia_mwait = { "mwait", "mwait", XX, XX, XX },
 Ia_negb_Eb = { "neg", "negb", Eb, XX, XX },
-Ia_negl_Ed = { "neg", "negl", Ed, XX, XX },
-Ia_negq_Eq = { "neg", "negq", Eq, XX, XX },
+//Ia_negl_Ed = { "neg", "negl", Ed, XX, XX },
+//Ia_negq_Eq = { "neg", "negq", Eq, XX, XX },
 Ia_negw_Ew = { "neg", "negw", Ew, XX, XX },
 Ia_nop = { "nop", "nop", XX, XX, XX },
 Ia_notb_Eb = { "not", "notb", Eb, XX, XX },
-Ia_notl_Ed = { "not", "notl", Ed, XX, XX },
-Ia_notq_Eq = { "not", "notq", Eq, XX, XX },
+//Ia_notl_Ed = { "not", "notl", Ed, XX, XX },
+//Ia_notq_Eq = { "not", "notq", Eq, XX, XX },
 Ia_notw_Ew = { "not", "notw", Ew, XX, XX },
 Ia_orb_AL_Ib = { "or", "orb", AL, Ib, XX },
 Ia_orb_Eb_Gb = { "or", "orb", Eb, Gb, XX },
@@ -271,14 +271,14 @@ Ia_pushfq = { "pushfq", "pushfq", XX, XX
 Ia_pushfw = { "pushf", "pushf", XX, XX, XX },
 Ia_pushl_CS = { "push", "pushl", CS, XX, XX },
 Ia_pushl_DS = { "push", "pushl", DS, XX, XX },
-Ia_pushl_Ed = { "push", "pushl", Ed, XX, XX },
+//Ia_pushl_Ed = { "push", "pushl", Ed, XX, XX },
 Ia_pushl_ES = { "push", "pushl", ES, XX, XX },
-Ia_pushl_Id = { "push", "pushl", Id, XX, XX },
+//Ia_pushl_Id = { "push", "pushl", Id, XX, XX },
 Ia_pushl_sIb = { "push", "pushl", sIbd, XX, XX },
 Ia_pushl_SS = { "push", "pushl", SS, XX, XX },
-Ia_pushq_Eq = { "push", "pushq", Eq, XX, XX },
-Ia_pushq_sIb = { "push", "pushq", sIbq, XX, XX },
-Ia_pushq_sId = { "push", "pushq", sIdq, XX, XX },
+//Ia_pushq_Eq = { "push", "pushq", Eq, XX, XX },
+//Ia_pushq_sIb = { "push", "pushq", sIbq, XX, XX },
+//Ia_pushq_sId = { "push", "pushq", sIdq, XX, XX },
 Ia_pushw_CS = { "push", "pushw", CS, XX, XX },
 Ia_pushw_DS = { "push", "pushw", DS, XX, XX },
 Ia_pushw_ES = { "push", "pushw", ES, XX, XX },
